exam_date/stored_date.py

```diff
@@ -0,0 +1,72 @@
+from autologging import logged, traced
+from pathlib import Path
+from datetime import datetime
+
+
+@logged
+@traced
+class AssignmentLatestSubmittedDate():
+    def __init__(self, path: str, file_name: str):
+        self.path = path
+        self.file_name = file_name
+
+    def _create_directory_if_not_exit(self) -> None:
+        """
+        creates a directory if doesn't exit with permission 0o777
+        :return:
```

Comment:
The `:return:` is unnecessary in the docstring, since the function is defined as returning `None`.  However, you could update the docstring of this function and the others to define that they raise exceptions.  Python typing doesn't provide a way to define exceptions thrown, so it should be done in the docstring.

Reply:
Removed it, I wasn't sure if I have to have a type hinting that is returning nothing. your comment is helpful in understanding that. I might go through the typing module little details for understanding. I have added docstring that it will raise exception if `dir` is not created